CardanoSharp.Wallet/Extensions/Models/Transactions/TransactionExtensions.cs

```diff
@@ -33,6 +33,9 @@ public static CBORObject GetCBOR(this Transaction transaction)
                 cborTransaction.Add(CBORObject.NewMap());
             }
 
+            //add isValid
+            cborTransaction.Add(transaction.IsValid.GetCBOR());
```

Comment:
transaction.IsValid is type "bool" which can never be null. Unless we want the type to be bool? so that it could be null?